library/components/TUrlCopyButton.vue

```diff
@@ -28,8 +28,11 @@ export default {
     url() {
       if (this.parentUrl) {
         this.parentUrl.searchParams.set("context[page]", this.page.url);
-        for (const [filter, value] of Object.entries(this.filters)) {
-          this.parentUrl.searchParams.set(filter, encodeURIComponent(value));
+        for (const filter of this.filters) {
+          this.parentUrl.searchParams.set(
+            filter.name,
+            encodeURIComponent(filter.value)
+          );
```

Comment:
This can be a `.map` instead. I find `Array.prototype.map` more friendly than a `for...of` but this is completely subjective 😄

Reply:
Huh. Never would have thought of using a .map there but darned if it isn't easier to read. :joy: